en/tutorials-and-examples/bookmarks/intro.rst

```diff
@@ -396,7 +396,7 @@ put the following content::
 
     <h1>
         Bookmarks tagged with
-        <?= $this->Text->toList($tags) ?>
+        <?= $this->Text->toList(h($tags)) ?>
```

Comment:
It works. But I didn't notice that `$tags` was an array 😛  If we are planning to remove the array handling feature from `h()` in the future, I would like to change the order. But it is a normal feature, this would be an example of usage.